whois-commons/src/main/java/net/ripe/db/whois/common/override/OverrideCredentialValidator.java

```diff
@@ -30,7 +30,12 @@ public boolean isAllowedAndValid(final boolean isTrusted, final UserSession user
             return false;
         }
         final OverrideCredential.OverrideValues overrideValues = overrideCredential.getOverrideValues().get();
-        return isAllowedToUseOverride(isTrusted, userSession, overrideValues.getUsername()) && isValidOverride(overrideValues, objectType);
+
+        try {
+            return isAllowedToUseOverride(isTrusted, userSession, overrideValues.getUsername()) && isValidOverride(overrideValues, objectType);
```

Comment:
Can you catch the exception in filterAuthFunction? As we do not catch exception for other public method and let the calling method handles it , it would be consistent

Reply:
OK, Done